mreg/models.py

```diff
@@ -327,6 +327,20 @@ def loc_string(self, zone):
         return '{name:30} IN {record_type:6} {record_data}\n'.format_map(data)
 
 
+class Sshfp(models.Model):
+    host = models.ForeignKey(Host, on_delete=models.CASCADE, db_column='host')
+    ttl = models.IntegerField(blank=True, null=True, validators=[validate_ttl])
+    algorithm = models.IntegerField(choices=((1, 'RSA'), (2, 'DSS'), (3, 'ECDSA'), (4, 'Ed25519')))
+    hash_type = models.IntegerField(choices=((1, 'SHA-1'), (2, 'SHA-256')))
+    fingerprint = models.CharField(max_length=64, validators=[validate_hexadecimal])
+
+    class Meta:
+        db_table = 'sshfp'
+
+    def __str__(self):
+        return str(self.host)
```

Comment:
Better __str__? Now all objects with the same host will be printed equally.

Reply:
Sorry, don't get it. the function name should be str?

Reply:
Wrong usage of github-markup. Should say better `__str__`. Something like:
`return "f{self.host.name} -> {self.algorithm} ({self.get_algorithm_display()}) {self.hash_type} ({self.get_hash_type_display()}) {self.fingerprint}"`

Though wrap/make it shorter.